bin/sirius-script-rabpm-mngr.py

```diff
@@ -61,8 +61,28 @@ def run_boot_from_flash(rack, slot):
 
 
 def add_rack_arg(command):
-    command.add_argument("-r", "--rack", required=True,
-                         help="Rack number {01..20} or 21 for transport line")
+    command.add_argument("-r", "--racks", required=True,
+                         nargs="+", action=ValidateDeviceListAction, all_devices=list(range(1, 22)),
+                         help="Rack numbers {01..21} (21 for transport line) or 'all'")
+
+
+class ValidateDeviceListAction(argparse.Action):
+    def __init__(self, all_devices, *args, **kwargs):
+        self._all_devices = all_devices
+        super(ValidateDeviceListAction, self).__init__(*args, **kwargs)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        if len(values) == 1 and values[0] == "all":
+            values = self._all_devices
+
+        else:
+            try:
+                values = list(map(int, values))
+            except ValueError as err:
+                parser.print_help()
+                parser.exit(
+                    status=1, message=f"Invalid rack argument: {err}\n")
```

Comment:
Speaking of generic, "Invalid rack" should be made more generic, or "rack" should be a parameter to the constructor.